doc/jinja-filters.md

```diff
@@ -0,0 +1,445 @@
+('', False, None, Undefined())  "False"
+```
+
+
+## format_household_name
+Concatenates a list of names into a single space separated string.
+
+##### Parameters: 
+- names: List of names to concatenate
+
+##### Context:
+To display a person's full name when the answers are entered as first, middle and last name. 
+```
+Are you <em>{{ [answers['first-name'][group_instance], answers['middle-names'][group_instance], answers['last-name'][group_instance]] | format_household_name }}<em>?
+```
+
+##### Examples:
+```
+Input:              Output:
+["John", "Doe"]     "John Doe"
+["John", ""]        "John"
+["", "Doe"]         "Doe"
+```
+
+
+## format_household_name_possessive
+Concatenates a list of names into a single space separated string and adds the possessive to the end (’s or just ’).
+
+##### Parameters: 
+- names: List of names to concatenate
+
+##### Context:
+To display a person's full name when the answers are entered as first, middle and last name but also to express a person's possession within the question. 
+```
+What is {{[answers['first-name'][group_instance], answers['last-name'][group_instance]] | format_household_name_possessive }} date of birth?
+```
+
+##### Examples:
+```
+Input:              Output:
+["John", "Doe"]     "John Doe's"
+["John", "Jones"]   "John Jones'"
+```
+
+## format_unordered_list
+Takes the first list from a list and wraps the items of that list in "ul" and "li" tags.
+
+##### Parameters: 
+- list_items: List of a list of items to be wrapped into a html list
+
+##### Context:
+Lists selected pay patterns for MWSS. Used as a description: 
+```
+{{ answers['pay-pattern-frequency-answer']|format_unordered_list }}
+```
+
+##### Examples:
+```
+Input:                  Output:
+[['item 1', 'item 2']]  "<ul><li>item 1</li><li>item 2</li></ul>"
+```
+
+## format_unordered_list_missing_items
+Will take a list of items and a list of possible items and return the possible items that are not in the initial list.
+
+##### Parameters: 
+- possible_items: All possible items 
+- list_items: Items actually present.
+
+##### Context:
+Used to return items that have not been selected from a Checkbox in e-commerce. Used as a description.
+```
+{{ format_unordered_list_missing_items(['Online ordering or reservation/booking', 'Description of goods or services, price lists', 'Order tracking', 'The possibility for visitors to customise or design the goods or services online', 'Personalised content for regular/repeat visitors', 'Links or references to this business’ social media profiles'], answers['answer1836']) }}
+```
+
+##### Examples:
+```
+possible_items = ['item 1', 'item 2', 'item 3', 'item 4']
+list_items = [['item 1', 'item 3']]
+
+Input:                          Output:
+(possible_items, list_items)    "<ul><li>item 2</li><li>item 4</li></ul>"
+```
+
+
+## format_household_summary
+A merge of format_household_name and format_unordered_list in order to return a list of full names in "ul" and "li' tags 
```

Comment:
If you got some free time on your hands, feel free to highlight any variable/function names in sentences. 

A merge of `format_household_name` and `format_unordered_list` in order to return a list of full names in "ul" and "li' tags

Reply:
No I need you to point them all out.